### DISCUSSION TITLE
Copilot is not reading the previous cells of the code and is generating code as if there were no cells before.

### DISCUSSION THREAD

**Discussion by FelixStaff** [community/community#63325] [Copilot Conversations]
The copilot is not reading the previous cells of my jupyter notebook codes and generates code as if it were a different file, it also does not check files.
![image](https://github.com/community/community/assets/94182561/c33ef38d-d64c-4d4d-ad69-6839eff6f3ff)

Here you can see how I ask it to use a class defined in a previous cell and it ignores it, it tells I ask that it use the bag class and it uses a class that does not exist called "Mochila"

Details: 
Copilot version: v1.101.317
Jupyter notebook version: v2023.7.1002162226

**Comment by alexlach** [comment]
I'm having the same problem, really making it useless in notebooks.
